packages/xo-web/src/common/xo/index.js

```diff
@@ -3082,7 +3082,10 @@ export const selfBindLicense = ({ id, plan }) =>
     },
     icon: 'unlock',
   })
-    .then(() => _call('xoa.licenses.bindToSelf', { licenseId: id }), noop)
+    .then(
+      () => _call('xoa.licenses.bindToSelf', { licenseId: id, oldXoaId }),
```

Comment:
The `oldXoaId` is picked from the license, so i don't think it's required to pass it under the chain call of methods.

Reply:
True, that's technically possible. I think that's still better to keep a parameter to make sure that a *rebind* was intended (instead of a normal bind). Do you think passing a `rebind` boolean would be better?

Reply:
Also it might be a good thing to pass the old XOA ID to ensure that the client side is aware that it's going to unbind that specific XOA and not another one in case of race condition.